rest/server_context.go

```diff
@@ -1020,7 +1020,7 @@ func (sc *ServerContext) Database(name string) *db.DatabaseContext {
 // If guest user defined, but has no access to channels .. issue warning + tips to fix
 func emitAccessRelatedWarnings(config *DbConfig, context *db.DatabaseContext) {
 	for _, warning := range collectAccessRelatedWarnings(config, context) {
-		base.Warn("%v", warning)
+		base.WarnR("%v", warning)
```

Comment:
Accidentally missed tagging warning? Or accidentally changed the logging function?

Reply:
Oops.  After converting this to `WarnR()` I noticed the code was dead and opened another PR, and forgot to circle back to this.